kik_unofficial/utilities/parsing_utilities.py

```diff
@@ -39,6 +39,17 @@ def get_text_safe(element: BeautifulSoup, tag: str) -> Union[str, None]:
     return element.text if element else None
 
 
+def get_attribute_safe(element: Tag, key: str) -> Union[str, None]:
```

Comment:
Maybe call this `get_optional_attribute`?

Reply:
It matches get_text_safe which is above. We can leave it the same or rename that also to `get_optional_text`

I prefer how it is now but if you absolutely think `get_optional_attribute` is better, I will change it.

Reply:
I think `optional` is better, because we're getting fields that could be missing, not really sanitizing input or something. Also maybe `get_text_safe` should be called `get_text_of_tag`